src/components/file-upload/FileUpload.tsx

```diff
@@ -115,6 +131,7 @@ export const FileUpload = ({
   onRemove,
   uploadedFile,
   width,
+  dropzoneDisabled,
```

Comment:
What do you think about using `disabled` to stay consistent with other amino components?
```suggestion
  disabled,
```

Reply:
That was my original intention. I thought about that, but then I think it will be confused with `disabled` props in dropzoneOptions. I want `disabled` prop only apply on the dropzone when there is no file selected. That's why I try the name to be more specific 🤔 What do you think?

Reply:
I'm confused. Let's chat about this.

Reply:
We chatted and determined that `dropzoneDisabled` is a good name, however we are removing the `dropzoneOptions.disabled` so we don't have two props disabling dropzone.